iarrc/src/color_detector/color_detector.cpp

```diff
@@ -40,41 +44,45 @@ namespace iarrc {
 
         const Mat &frameBGR = cv_ptr->image;
         Mat frameBlurred;
-        GaussianBlur(frameBGR, frameBlurred, Size{7,7}, 0);
+        GaussianBlur(frameBGR, frameBlurred, Size{0,0}, 7);
```

Comment:
Just a note/FYI for OpenCV users. The last parameter of GaussianBlur is the standard deviation of the distribution that populates the linear filter. So before this change, there was not gaussian filtering going on. The size argument specifies the shape of the linear filter, and if it's (0,0) then the filter will be automatically sized based on the standard deviation.